src/main/scala/chisel3/stage/ChiselAnnotations.scala

```diff
@@ -399,3 +400,54 @@ case object UseLegacyWidthBehavior
     )
   )
 }
+
+/** This records a mapping from an old [[chisel3.layer.Layer]] to a new [[chisel3.layer.Layer]].
+  *
+  * This is intended to be used by a downstream Chisel project that is using an
+  * upstream Chisel project which has different layers and the user would like
+  * to align the upstream project with the downstream.
+  */
+case class RemapLayer(oldLayer: Layer, newLayer: Layer) extends NoTargetAnnotation with ChiselOption with Unserializable
+
+object RemapLayer extends HasShellOptions {
+
+  private val re = "([\\w\\$.]+),([\\w\\$.]+)".r
```

Comment:
```suggestion
  private val Remap = "([\\w\\$.]+),([\\w\\$.]+)".r
```
Nit but IMO this should be UpperCamelCase and some sort of more meaningful name to make the use below more clear.

Reply:
I'm not used to seeing `val`s with upper camel case. However, I'll make this more descriptive and move it closer to its use site. Changed in: 167b1bebe